packages/client/lib/rpc/modules/engine.ts

```diff
@@ -522,4 +533,23 @@ export class Engine {
       }
     }
   }
+
+  /**
+   * Sets and returns the transition configuration parameters.
+   *
+   * @param params An array of one parameter:
```

Comment:
@ryanio this api doesn't sets the configuration, it just matches and logs (on a mismatch) and nothing else: https://github.com/ethereum/execution-apis/blob/f6fdc57ccb0ae3dd82812993a84e9fb12ebff798/src/engine/specification.md#engine_exchangetransitionconfigurationv1 
keyword being *surface*, also responds with its own user set params so that CL can also log mismatch if any.

Reply:
ooh that makes sense, thanks :) updated in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1750/commits/851378cc0f2e980f542cf08172e4351ec7843ecf